tests/observability/metrics/conftest.py

```diff
@@ -1046,3 +1047,23 @@ def deleted_ssp_operator_pod(admin_client, hco_namespace):
 @pytest.fixture(scope="class")
 def initiate_metric_value(request, prometheus):
     return get_metrics_value(prometheus=prometheus, metrics_name=request.param)
+
+
+@pytest.fixture()
+def vm_for_vm_disk_allocation_size_test(namespace, admin_client):
```

Comment:
We need to remove cirros from the tests, so please use something else. 
You can use `Images.Fedora.FEDORA_CONTAINER_IMAGE`, or clone from a DataSource:
```
    with VirtualMachineForTests(
        client=admin_client,
        name="fedora-vm",
        namespace=namespace.name,
        data_volume_template=data_volume_template_with_source_ref_dict(
            data_source=DataSource(
                name=OS_FLAVOR_FEDORA,
                namespace=golden_images_namespace.name
            ),
            storage_class=py_config["default_storage_class"]
        ),
        memory_guest=Images.Fedora.DEFAULT_MEMORY_SIZE,
    ) as vm:

```